### DISCUSSION TITLE
Is there a way to use different attachment folders for each note?

### DISCUSSION THREAD

**Discussion by calofmijuck** [ObsidianPublisher/obsidian-github-publisher#220] [Q&A — Plugin] [answered]
As far as I know, it seems that the attachment folder is set globally, throughout the vault.

Is there a way to customize the attachment folder for each note?
For instance, a property key to set the attachment folder?
(It seems that sending notes to different folders is possible)

Thanks for the great plugin!

**Comment by Mara-Li** [comment, marked as answer]
Hello. Yes, it is possible using property key. Each key are described in the help section thought the plugin settings.

**Reply by calofmijuck** [reply]
Thanks, I didn't know the key was for this purpose.
